geometry_msgs/Point obj1position
geometry_msgs/Point obj2position